packages/flutter_tools/test/general.shard/macos/xcode_validator_test.dart

```diff
@@ -182,6 +183,11 @@ void main() {
       final XcodeValidator validator = XcodeValidator(xcode: xcode, userMessages: UserMessages());
       final ValidationResult result = await validator.validate();
       expect(result.type, ValidationType.installed);
```

Comment:
@jmagman 
This was accidentally removed, restored it cb7aa1b65f8dba5a8a47b681c0b45c05577263b8